crates/rooch/src/commands/env/commands/add.rs

```diff
@@ -18,14 +18,23 @@ pub struct AddCommand {
     pub rpc: String,
     #[clap(long, value_hint = ValueHint::Url)]
     pub ws: Option<String>,
+    #[clap(long, value_hint = ValueHint::Url)]
+    pub proxy: Option<String>,
```

Comment:
The proxy is global, not a configuration option for Rooch Env.

Reply:
> The proxy is global, not a configuration option for Rooch Env.

I think it should and shall be distinguished from the global proxy for each specific application. Global proxy in the system and the proxy in the application are two different elements.